dts/arm/arduino_due.dts

```diff
@@ -0,0 +1,22 @@
+/dts-v1/;
+
+#include <atmel/sam3x.dtsi>
+
+/ {
+	model = "Arduino Due with an Atmel SAM3X8E";
+	compatible = "atmel,sam3x";
```

Comment:
Is the ordering important here, even if by convention? From most specific to the most generic it would be `compatible = "arduino,due", "atmel,sam3x8e", "atmel,sam3x";` then.